Source/Extensions/Blazorise.Snackbar/Utils/Extensions.cs

```diff
@@ -6,11 +6,21 @@ public static string GetName( this SnackbarLocation snackbarLocation )
     {
         switch ( snackbarLocation )
         {
+            case SnackbarLocation.Top:
+                return "top";
+            case SnackbarLocation.TopStart:
+                return "top-left";
+            case SnackbarLocation.TopEnd:
+                return "top-right";
```

Comment:
Can we have start/end support? Or does it not make sense here?

Reply:
if i understand, you think that placing the snackbar in the middle of the screen on the left or right side is what can be added?... If so, i don't think it makes sense because the snackbar animations work by showing it from the bottom or top of the screen, possibly add an animation like offcanvas to slide on left or right side of the screen

Reply:
Agree, no need to make it at center left/right.

Reply:
I was asking about the css class naming. If it needs to be left right instead of start end